services/web/server/src/simcore_service_webserver/director_v2_core.py

```diff
@@ -116,9 +116,7 @@ async def _request_director_v2(
                     # - `sometimes director-v0` (via redirects) replies
                     #   in plain text and this is considered an error
                     # - `director-v2` and `director-v0` can reply with 204 no content
-                    if response.status != expected_status.status_code or isinstance(
-                        payload, str
-                    ):
+                    if response.status != expected_status.status_code:
```

Comment:
well for what I care, I was not the one that added this in the first place. BUT, when I read the comment above:
1. they do not fit with what happens anymore (which is why I think too many comments are usually bad because no one reads them and most of the time they are then wrong and misleading).
2. So maybe check with @pcrespov if he was the one that added this?

Reply:
No, I was the one adding it. I will check with him, but for now will keep it removed. @pcrespov

Reply:
then at least remove the comments.